test/check-verify

```diff
@@ -152,6 +167,8 @@ def main():
             # Get the relevant commit
             subprocess.check_call([ "git", "fetch", "origin", ref ])
             subprocess.check_call([ "git", "checkout", revision ])
+            if ref == "master":
```

Comment:
I think this logic can go into start_publishing(), and just be in one place. We have the @name parameter there.